Obsidian/Utils/WADHashGenerator.cs

```diff
@@ -175,12 +174,13 @@ private static IEnumerable<string> ProcessBINLinkedFiles(List<string> linkedFile
         {
             List<string> strings = new List<string>();
 
-            for (int i = 0; i < linkedFiles.Count; i++)
+            foreach (string fetchedString in linkedFiles)
             {
-                string fetchedString = linkedFiles[i];
-
                 strings.Add(fetchedString);
-                strings.AddRange(ProcessBINPackedLinkedFile(fetchedString));
+                if (fetchedString.Contains("_skins_skin"))
```

Comment:
change to _Skins_Skin as string.Contains is case sensitive

Reply:
also add '|| fetchedString.Contains("_Skins_Root")'

Reply:
> also add '|| fetchedString.Contains("_Skins_Root")'

Shouldn't be necessary, as every string that contains ``_skins_root`` will also contain the string of at least one other skin, as no "bundled" files with only one skin exist.